owncloudApp/src/main/java/com/owncloud/android/presentation/files/filelist/FileListAdapter.kt

```diff
@@ -285,7 +293,7 @@ class FileListAdapter(
                 // Set file icon depending on its mimetype. Ask for thumbnail later.
                 fileIcon.setImageResource(MimetypeIconUtil.getFileTypeIconId(file.mimeType, file.fileName))
                 if (file.remoteId != null) {
```

Comment:
This condition makes no sense anymore if we are not using the `file.remoteId` here. Maybe it should be moved together with the initialization of the `thumbnail` variable